src/lib/i18n/texts/de.json

```diff
@@ -182,23 +182,23 @@
     "herald": "Herold"
   },
   "words": {
-    "played": "Played",
-    "times": "times",
-    "ratio": "Ratio",
-    "rank": "Rank",
+    "played": "Gespielt",
+    "times": "mal",
+    "ratio": "Verhältnis",
+    "rank": "Rang",
     "optional": "Optional",
-    "dismiss": "Dismiss",
-    "embed": "Embed",
-    "loadMore": "Load More"
+    "dismiss": "Verwerfen",
+    "embed": "Einbetten",
+    "loadMore": "Mehr laden"
   },
   "leaderboards": {
-    "name": "Leaderboards"
+    "name": "Rangliste"
   },
   "messages": {
     "gw2ApiDown": "The official GW2 API is experiencing issues, your session may be affected!",
```

Comment:
Yeah, it somehow slipped through. Changed it now. Do i need to resend the pull request or is it updated automatically when I commit the changes?